modules/browser_object_tracker_panel.py

```diff
@@ -15,6 +15,58 @@ class TrackerPanel(BasePage):
 
     URL_TEMPLATE = ""
 
+    def verify_allowed_blocked_trackers(
```

Comment:
This is great, but I think it would be more useful if both `allowed` and `blocked` had default empty set values, that way we could use this for all three of: verifying that allowed trackers are allowed, verifying that blocked trackers are blocked, and both at the same time. Currently, this just does both. If there are other methods that do one or the other, they could be replaced by this in the BOM and the tests.